app/views/brexit_checker/_action_list.html.erb

```diff
@@ -1,7 +1,7 @@
 <% actions.each.with_index do | action, index | %>
   <div class="govuk-grid-row brexit-checker__action">
     <div class="brexit-checker__content-wrapper">
-      <p class="govuk-body brexit-checker__action_title">
+      <h4 class="govuk-body brexit-checker__action_title">
```

Comment:
Looks like this should be a H3? (I think it's this one, there's definitely a H4 somewhere in this page that should be a H3).

<img width="573" alt="Screenshot 2020-09-24 at 16 05 33" src="https://user-images.githubusercontent.com/861310/94163516-dbca6e80-fe7f-11ea-9010-493fd432a5ef.png">

Reply:
(ignore the H2/cookies error at the top, that's actually okay)

Reply:
Ah, that's a snag. The business section doesn't get the h3 title that the other bits get 😞 . I'll have to have a wrestle with that. I don't really want to promote all the business actions to be h3s, though cos I reckon all the actions should remain at the same level. 
Humph

Reply:
And boo

Reply:
Sorry...

Reply:
OK, I've raised this with product/content to hopefully get an appropriate h3 for the business section.

Reply:
I've added a commit to use h3s on business results and h4s on citizen results. The commit has more details about the longer term options which would be better.